dhall/tests/Dhall/Test/TypeInference.hs

```diff
@@ -89,6 +89,7 @@ failureTest prefix = do
                , typeInferenceDirectory </> "failure/unit/RecordTypeDuplicateFields"
                , typeInferenceDirectory </> "failure/unit/UnionTypeDuplicateVariants1"
                , typeInferenceDirectory </> "failure/unit/UnionTypeDuplicateVariants2"
+               , typeInferenceDirectory </> "failure/unit/RecordProjectionDuplicateFields"
```

Comment:
I had already added this on line 88. Sorry for the duplicate work! :/

Reply:
No worries! 🙂